assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -1936,6 +1938,16 @@ describe( 'core/modules modules', () => {
 					}
 				);
 
+				fetchMock.getOnce(
+					new RegExp(
+						'^/google-site-kit/v1/modules/analytics/data/settings'
+					),
+					{
+						body: {},
+						status: 200,
+					}
+				);
```

Comment:
Since we don't care about the response in this particular test case, we can probably replace the mock with a `muteFetch`

```suggestion
				muteFetch(
					new RegExp(
						'^/google-site-kit/v1/modules/analytics/data/settings'
					)
				);
```

Reply:
Updated, thank you!